Sources/XcbeautifyLib/String+CapturedGroups.swift

```diff
@@ -391,7 +391,7 @@ extension String {
         case .linkerDuplicateSymbolsLocation:
             assert(results.count >= 1)
             guard let reason = results[safe: 0] else { return EmptyCaptureGroup() }
-            return LinkerDuplicateSymbolsCaptureGroup(reason: reason)
+            return LinkerDuplicateSymbolsLocationCaptureGroup(filePath: reason)
```

Comment:
Looks like there's a mismatch between the parameter and the variable. Can you update to reflect the actual expected value?

Reply:
OK.